agent/api/testutils/container_equal_test.go

```diff
@@ -20,12 +20,6 @@ import (
 )
 
 func TestContainerEqual(t *testing.T) {
```

Comment:
This should also be a good opportunity to turn this into [table-driven tests](https://blog.golang.org/subtests). Example here: https://github.com/aws/amazon-ecs-agent/blob/master/agent/engine/task_manager_test.go#L36